docs/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title> Check-it-Out </title>
+    <!-- Import Github-Markdown -->
+    <link rel="stylesheet" href="github-markdown.css">
+    <!-- Import latos font -->
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,900" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./main.css">
+</head>
+<body>
+    <header>
+        <h1 class="headline"> Check It Out </h1>
+        <h2 class="subheadline"> Less Keystrokes. More Work Done.</h2>
+        <div class="link-container">
+            <a href="https://badge.fury.io/js/check-it-out"><img src="https://badge.fury.io/js/check-it-out.svg" alt="npm version" height="18" class="npm-badges"></a>
+            <a href="https://badge.fury.io/gh/jwu910%2Fcheck-it-out"><img src="https://badge.fury.io/gh/jwu910%2Fcheck-it-out.svg" alt="GitHub version" class="npm-badges"/></a>
+        </div>
+    </header>
+    <div class="main">
+        <div class="info-section">
+            <h2 class="title"> What is Check It Out? </h2>
+            <p class="contents"> Check it out, before you checkout. </p>
+            <p class="contents"> Too many keystrokes just to checkout a branch? Tired of long branch names? Check it out gives you the interactive power to see your branches and check them out with fewer keystrokes </p>
+        </div>
+        <div class="require-section">
+            <h2 class="title"> Requirements </h2>
+            <p class="require-links"><a href="https://nodejs.org/en/blog/release/v7.0.0/">Node >= v7.0 </a> - Required for async/await </p>

```

Comment:
Heads up. I fixed this node 7. requirement and it will change on the readme soon. Haven't tested what minimum version will be required.